budgetportal/tests/helpers.py

```diff
@@ -45,3 +46,41 @@ def wait_until_text_in(self, selector, text):
         self.wait.until(
             EC.text_to_be_present_in_element((By.CSS_SELECTOR, selector), text)
         )
+
+    def _fixture_teardown(self):
+        # Allow TRUNCATE ... CASCADE and don't emit the post_migrate signal
+        # when flushing only a subset of the apps
+        for db_name in self._databases_names(include_mirrors=False):
+            # Flush the database
+            inhibit_post_migrate = (
+                    self.available_apps is not None or
+                    (  # Inhibit the post_migrate signal when using serialized
+                        # rollback to avoid trying to recreate the serialized data.
+                            self.serialized_rollback and
+                            hasattr(connections[db_name], '_test_serialized_contents')
+                    )
+            )
+            call_command('flush', verbosity=0, interactive=False,
+                         database=db_name, reset_sequences=False,
+                         allow_cascade=True,
+                         inhibit_post_migrate=inhibit_post_migrate)
+
+
+class CustomLiveServerTestCase(LiveServerTestCase):
```

Comment:
Its probably not needed as tests that used `LiveServerTestCase` could be simple APITestCase IMHO.
But lets leave it as the last part. It was really time-consuming to discover those things.